parser/src/test/java/com/facebook/coresql/lint/TestMixedAndOr.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.coresql.lint;
+
+import com.facebook.coresql.parser.AstNode;
+import com.facebook.coresql.warning.DefaultWarningCollector;
+import com.facebook.coresql.warning.WarningCollectorConfig;
+import org.testng.annotations.Test;
+
+import static com.facebook.coresql.parser.ParserHelper.parseStatement;
+import static org.testng.Assert.assertEquals;
+
+public class TestMixedAndOr
+{
+    private static final LintingVisitor lintingVisitor = new MixedAndOr(new DefaultWarningCollector(new WarningCollectorConfig().setMaxWarnings(1)));
```

Comment:
In general, constants in Java use all uppercase with underscores. So LINTING_VISITOR instead of lintingVisitor.

Reply:
DONE